sched/sched/sched_timerexpiration.c

```diff
@@ -205,8 +205,8 @@ static uint32_t nxsched_cpu_scheduler(int cpu, uint32_t ticks,
        * committed to updating the scheduler for this TCB.
        */
 
-      sporadic->sched_time.tv_sec  = g_sched_time.tv_sec;
-      sporadic->sched_time.tv_nsec = g_sched_time.tv_nsec;
+      sporadic->eventtime = USEC2TICK(1000000 * (uint64_t)g_sched_time.tv_sec
```

Comment:
```suggestion
      sporadic->eventtime = USEC2TICK(USEC_PER_SEC * (uint64_t)g_sched_time.tv_sec
```

Reply:
Or maybe even use `sporadic->eventtime = SEC2TICK(g_sched_time.tv_sec) + NSEC2TICK(g_sched_time.tv_nsec)`?

Reply:
I'm so sorry for my mistake,thanks very much for your help!